src/@batch-flask/ui/virtual-scroll/virtual-scroll.component.ts

```diff
@@ -85,17 +85,17 @@ export class VirtualScrollComponent implements OnInit, AfterViewInit, OnChanges,
 
     public viewportItems: any[] = [];
 
-    @ViewChild("content", { read: ElementRef })
+    @ViewChild("content", { read: ElementRef, static: true })
     public contentElementRef: ElementRef;
 
-    @ContentChild(VirtualScrollTailComponent)
+    @ContentChild(VirtualScrollTailComponent, {static: true})
     public set tail(tail: VirtualScrollTailComponent) {
         this._tail = tail;
         this.refresh();
     }
     public get tail() { return this._tail; }
 
-    @ContentChild(VirtualScrollRowDirective) public rowDef: VirtualScrollRowDirective<any>;
+    @ContentChild(VirtualScrollRowDirective, {static: true}) public rowDef: VirtualScrollRowDirective<any>;
```

Comment:
nit: spaces around {}

Reply:
You could look into adding `prettier` to format and `tslint-prettier` to enforce the syntax. As it didn't use it before it will be a massive fix most likely to work with its formatting but might be worth it to elimiate style issues

Reply:
Okay I'll take a look into `prettier` and `tslint-prettier`, sounds helpful!